data-prepper-expression/src/main/java/org/opensearch/dataprepper/expression/OperatorConfiguration.java

```diff
@@ -17,7 +17,7 @@
 import java.util.function.BiPredicate;
 
 @Named
-public class OperatorFactory {
+public class OperatorConfiguration {
```

Comment:
Nit: we can make it package private

Reply:
Good find! In one of the other PRs I moved a few classes in this package to package private. I'll make sure this class is on the list